client/karma.js

```diff
@@ -128,8 +128,10 @@ function Karma (updater, socket, iframe, opener, navigator, location, document)
       }
     // run in iframe
     } else {
+      iframe.onload = () => {
+        karmaNavigating = false
+      }
       iframe.src = policy.createURL(url)
```

Comment:
Immediately before this statement, `karmaNavigating` should be `true`. This statement sets the iframe URL.  Any action after this statement that causes an unload event is an error.  Thus we immediately set `karmaNavigating` to be `false`.

Reply:
Looks like we were using a really old version of chrome (71) where the order wasn't deterministic for some reason. Meaning it was actually triggering the unload after the flag was being set to false. Fixed it on our side by upgrading chrome. Thanks!